charmcraft/commands/store/store.py

```diff
@@ -207,7 +208,15 @@ def login(self, permissions=None, ttl=None, charms=None, bundles=None, channels=
         if packages:
             kwargs["packages"] = packages
 
-        return self._client.login(**kwargs)
+        try:
+            credentials = self._client.login(**kwargs)
+        except CredentialsAlreadyAvailable as exc:
+            raise CraftError(
+                "Cannot login because credentials were found in your system "
+                "(which may be no longer valid, though).",
+                resolution="Please logout before login again.",
```

Comment:
`Please logout before login again.` -> `Please logout before logging in again.` 

(or to keep "login" in simple tense: `Please logout first, then login again.`)